tools/regression.py

```diff
@@ -162,6 +162,10 @@ def execonf(file, build):
 	logging.getLogger().setLevel(getattr(logging, str(args.log).upper(), logging.WARN))
 
 	if args.inviwo:
+		if not os.path.exists(args.inviwo):
```

Comment:
Does this on also work with relative paths?

Reply:
it should.
https://docs.python.org/3/library/os.path.html#os.path.exists